live-examples/css-examples/padding.html

```diff
@@ -0,0 +1,43 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="padding">
```

Comment:
Same comment here as with `margin`

Reply:
As for `margin`, I can't remove this unless we change the editor logic.